addon/components/o-s-s/phone-number-input.ts

```diff
@@ -41,7 +41,7 @@ export default class OSSPhoneNumberInput extends Component<OSSPhoneNumberInputAr
       typeof this.args.onChange === 'function'
     );
 
-    this.selectedCountry = this._countries[0];
+    this.selectedCountry = this._countries[0]!;
```

Comment:
Is this throwing a warning on the new node version ? that's weird 🤔

Reply:
More like the TS version not letting it slide anymore. I think they fixed something because it was never supposed to let it slide IMO